### DISCUSSION TITLE
제페토 unity 오류 한번만 봐주세요 님들 ㅠㅠ

### DISCUSSION THREAD

**Discussion by DemainTrust** [naverz/zepeto-studio-kor#618] [Issue Report (Legacy)]
제페토 월드 스튜디오 템플릿을 적용하고 여는 과정인데

이렇게 몇시간간쨰 진행을 못하고 있어요 

어떻게해야 해결할 수 있을까요>?
![스크린샷 2022-04-14 오후 8 14 43](https://user-images.githubusercontent.com/87153682/163380156-5d60d1e3-20ad-46ed-a12e-6e4125485eca.png)
![스크린샷 2022-04-14 오후 8 14 43](https://user-images.githubusercontent.com/87153682/163380260-70a08d78-29c5-4d9b-96e9-16db7a6affc7.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 오류 메세지를 보았을때 새로운 프로젝트를 제페토 템플릿으로 생성하는 것이 아니라,
유니티 버전5 이하일때 제작한 프로젝트를 여는 것으로 보여집니다.
제페토 템플릿을 적용하는 과정을 아래 2개의 가이드를 따라서 다시 진행 부탁드리며, 가이드에 맞게해도 동일한 오류가 발생할 경우 말씀 주세요.

https://studio.zepeto.me/guides/installing-unity-and-registering-zepeto-template
https://studio.zepeto.me/guides/getting-started-with-templates
